In [20]:
from math import sqrt
import numpy as np

### 1
#### Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением, равным 16. Найти доверительный интервал для оценки математического ожидания с надежностью 0.95,если выборочная средняя M = 80, а объем выборки n = 256.

$X +- z_{\alpha/2} * \frac{\sigma}{\sqrt{n}}$

$z_{\alpha/2} = 1.96$

In [7]:
m = 80
n = 256
z_a = 1.96
sigma = sqrt(16)
int(sigma)

4

In [12]:
left_edge = m - z_a*sigma/sqrt(n)
right_edge = m + z_a*sigma/sqrt(n)

In [13]:
print(f'[{left_edge}:{right_edge}]')

[79.51:80.49]


#### Ответ: [79.51 : 80.49]

### 2
#### В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью, получены опытные данные: 6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1 Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей, оценить истинное значение величины X при помощи доверительного интервала, покрывающего это значение с доверительной вероятностью 0,95.

In [16]:
experements = [6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1]

In [26]:
n = len(experements)
m = sum(experements)/n
sigma = np.std(experements)
z_a = 1.96

In [27]:
left_edge = m - z_a*sigma/sqrt(n)
right_edge = m + z_a*sigma/sqrt(n)

In [29]:
print(f'[{left_edge} : {right_edge}]')

[6.324928568117949 : 6.855071431882049]


#### Ответ: [6.32 : 6.86]

### 3
#### Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм. Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметроказался равным 17.5 мм, а дисперсия известна и равна 4 кв.мм.

$H_0$ = Средний диаметр равен 17мм, $M_0 = M$

In [32]:
m0 = 17
m = 17.5
sigma = sqrt(4)
n = 100
z_a = 1.64

$z = \frac{M * m_0}{\sigma / \sqrt{n}}$

In [42]:
z = (m - m0) / (sigma / sqrt(n))
z

2.5

#### Ответ: рассчетное z оказался больше $z_a$, поэтому мы отвергаем нулевую гипотезу, средние диаметры шариков для подшипников отличаются

### 4
#### Продавец утверждает, что средний вес пачки печенья составляет 200 г. Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет: 202, 203, 199, 197, 195, 201, 200, 204, 194, 190. Известно, что их веса распределены нормально. Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

$H_0$ = Средний вес пачки = 200г, $M_0 = M$

In [51]:
experements = [202, 203, 199, 197, 195, 201, 200, 204, 194, 190]
m0 = 200
n = 10
z_a = 2.57
m = sum(experements) / len(experements)
sigma = np.std(experements)

In [54]:
z = (m - m0) / (sigma / sqrt(n))
z

-1.1227217828476797

#### Ответ: z оказался в промежутке [-2.57 : 2.57], не можем отвергнуть нулевую гипотезу